redis/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM redis:3.2.8
+COPY redis.conf /usr/local/etc/redis/redis.conf
+CMD [ "redis-server", "/usr/local/etc/redis/redis.conf" ]
```

Comment:
It shouldn't be necessary to override the CMD if we just copy our custom Redis config into the default location Redis will look at. That's /etc/redis.conf according to the Redis image readme.